### DISCUSSION TITLE
Build Error Messages

### DISCUSSION THREAD

**Discussion by nagymusic** [mmistakes/minimal-mistakes#4470] [General]
After migrating to a new MacBook Pro laptop, I started getting more elaborate error messages when building the site. Such as the one I include below. 

I would be grateful if someone could help me understand what's happening or point me in the right direction. Thank you!

zvony@MacBook-Pro nagymusic % JEKYLL_ENV=production bundle exec jekyll build
Configuration file: /Users/zvony/Library/Mobile Documents/com~apple~CloudDocs/Documents/nagymusic/_config.yml
To use retry middleware with Faraday v2.0+, install `faraday-retry` gem
            Source: /Users/zvony/Library/Mobile Documents/com~apple~CloudDocs/Documents/nagymusic
       Destination: /Users/zvony/Library/Mobile Documents/com~apple~CloudDocs/Documents/nagymusic/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
       Jekyll Feed: Generating feed for posts
Deprecation Warning: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div(9, 16)

More info and automated migrator: https://sass-lang.com/d/slash-div

   ╷
31 │ $mfp-iframe-ratio:                    9/16;                       // Ratio of iframe (9/16 = widescreen, 3/4 = standard, etc.)
   │                                       ^^^^
   ╵
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/vendor/magnific-popup/_settings.scss 31:39      @import
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/vendor/magnific-popup/_magnific-popup.scss 3:9  @import
    minimal-mistakes.scss 13:9                                                                                        @import
    /Users/zvony/Library/Mobile Documents/com~apple~CloudDocs/Documents/nagymusic/assets/css/main.scss 4:9            root stylesheet
Deprecation Warning: Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($value, 16px) or calc($value / 16px)

More info and automated migrator: https://sass-lang.com/d/slash-div

   ╷
28 │     @return $value / 16px * 1em;
   │             ^^^^^^^^^^^^^
   ╵
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/vendor/breakpoint/_helpers.scss 28:13                base-conversion()
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/vendor/breakpoint/_helpers.scss 20:13                breakpoint-to-base-em()
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/vendor/breakpoint/parsers/single/_default.scss 8:29  breakpoint-parse-default()
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/vendor/breakpoint/parsers/_single.scss 22:14         breakpoint-parse-single()
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/vendor/breakpoint/_parsers.scss 153:18               breakpoint-parse()
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/vendor/breakpoint/_parsers.scss 48:22                breakpoint()
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/vendor/breakpoint/_breakpoint.scss 46:16             breakpoint()
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/_reset.scss 13:3                                     @import
    minimal-mistakes.scss 18:9                                                                                             @import
    /Users/zvony/Library/Mobile Documents/com~apple~CloudDocs/Documents/nagymusic/assets/css/main.scss 4:9                 root stylesheet
Deprecation Warning: Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div(5px, 2) or calc(5px / 2)

More info and automated migrator: https://sass-lang.com/d/slash-div

   ╷
28 │     margin-bottom: (5px / 2);
   │                     ^^^^^^^
   ╵
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/_forms.scss 28:21                     @import
    minimal-mistakes.scss 20:9                                                                              @import
    /Users/zvony/Library/Mobile Documents/com~apple~CloudDocs/Documents/nagymusic/assets/css/main.scss 4:9  root stylesheet
Deprecation Warning: Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div(($red * 299) + ($green * 587) + ($blue * 114), 1000) or calc((($red * 299) + ($green * 587) + ($blue * 114)) / 1000)

More info and automated migrator: https://sass-lang.com/d/slash-div

   ╷
68 │   $yiq: (($red*299)+($green*587)+($blue*114))/1000;
   │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/_mixins.scss 68:9                     yiq-is-light()
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/_mixins.scss 81:14                    yiq-contrast-color()
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/_mixins.scss 91:10                    yiq-contrasted()
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/_buttons.scss 46:7                    @import
    minimal-mistakes.scss 25:9                                                                              @import
    /Users/zvony/Library/Mobile Documents/com~apple~CloudDocs/Documents/nagymusic/assets/css/main.scss 4:9  root stylesheet
Deprecation Warning: Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($span-width, $container) or calc($span-width / $container)

More info and automated migrator: https://sass-lang.com/d/slash-div

   ╷
93 │     @return percentage($span-width / $container);
   │                        ^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/vendor/susy/susy/_su-math.scss 93:24          su-span()
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/vendor/susy/susy/_syntax-helpers.scss 190:11  su-call()
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/vendor/susy/susy/_api.scss 146:13             susy-span()
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/vendor/susy/susy/_unprefix.scss 19:11         span()
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/_utilities.scss 185:24                        @content
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/vendor/breakpoint/_breakpoint.scss 66:7       breakpoint()
    ../../../../minimal-mistakes-jekyll-4.24.0/_sass/minimal-mistakes/_utilities.scss 184:3                         @import
    minimal-mistakes.scss 34:9                                                                                      @import
    /Users/zvony/Library/Mobile Documents/com~apple~CloudDocs/Documents/nagymusic/assets/css/main.scss 4:9          root stylesheet
Warning: 1 repetitive deprecation warnings omitted.
Run in verbose mode to see all warnings.
                    done in 3.052 seconds.
 Auto-regeneration: disabled. Use --watch to enable.

**Comment by positron96** [comment]
I have them too. These are warnings, seems to be working fine with them.
